tests/test-generate-context/choices_template.json

```diff
@@ -3,5 +3,6 @@
     "github_username": "hackebrot",
     "project_name": "Kivy Project",
     "repo_name": "{{cookiecutter.project_name|lower}}",
-    "orientation": ["all", "landscape", "portrait"]
+    "orientation": ["all", "landscape", "portrait"],
+    "is_alive": true
```

Comment:
I'd prefer to have a separate test file for booleans variables.

We might have a hard time debugging test failures. Can you please set up a dedicated test for booleans?